public/includes/find-house.php

```diff
@@ -0,0 +1,51 @@
+<?php
+require_once('db.php');
+
+function get_states() {
+    $mysqli = db_connection();
+    $sql_query = "SELECT * FROM `states`";
+    $result = mysqli_query($mysqli,$sql_query);
+    $json_response = array();
+    while ($row = mysqli_fetch_assoc($result)) {
+        $json_response[] = $row;
+    }
+    echo json_encode($json_response);
+    db_query_error($sql_query);
```

Comment:
The database query should be returned separately. You can use an if statement to check if there is an error and then return the error message. Note: It is bad practices to return the success response and error together.